### PR TITLE
changes to styling and white spaces

### PR DESCRIPTION
**Pull request by Dilyen**
Removed unneeded styling and made the code looks more nicer by removing possible unwanted white spaces  as well.

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -85,10 +85,10 @@
 
   </style>
```
Comment (YaaNuamah) [inline dist/index.html]: Please remove whitespaces and check indentations.

File: dist/index.html [hunk 2]
```diff
@@ -85,10 +85,10 @@
...
 
-<link rel="stylesheet" href="styles.4d523c3348d33c2010b3.css"></head>
+</head>
```
Comment (YaaNuamah) [inline dist/index.html]: please remove whitespaces.

File: src/app/dashboard/dashboard.component.html [hunk 3]
```diff
@@ -71,7 +71,9 @@ <h6 class="font-weight-medium text-right mb-0">Request
   <div class="col-lg-100 grid-margin">
     <div class="card">
       <div class="card-body">
+      <hr>
         <h4 class="card-title">Current Status</h4>
+        <hr>
```
Comment (YaaNuamah) [inline src/app/dashboard/dashboard.component.html:76]: indent properly.

File: src/app/dashboard/dashboard.component.ts [hunk 4]
```diff
@@ -16,27 +15,28 @@ export interface StatusData {
...
         project: "Pollster",
-        status: 500,
+        status: 200,
         endpoint: "https://employeeservice002.herokuapp.com/v1/api/employees",
-        requestmethod: ""
-    }
+        requestmethod: "GET"
+    },
+    {
+    project: "Chess",
```
Comment (YaaNuamah) [inline src/app/dashboard/dashboard.component.ts:33]: check indentation.

File: src/app/empire.service.spec.ts [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import { TestBed } from '@angular/core/testing';
+
```
Comment (YaaNuamah) [inline src/app/empire.service.spec.ts:2]: remove whitespaces.

File: src/app/status/status.component.ts [hunk 6]
```diff
@@ -23,8 +23,14 @@ export class StatusComponent implements OnInit {
         project: "Pollster",
         status: 500,
         endpoint: "https://employeeservice002.herokuapp.com/v1/api/employees",
-        requestmethod: ""
-    }
+        requestmethod: "GET"
+    },
+    {
+      project: "Chess",
```
Comment (YaaNuamah) [inline src/app/status/status.component.ts:29]: indent properly.

**Review by YaaNuamah — COMMENTED** [review]
Looking forward to changes.

File: dist/index.html [hunk 7]
```diff
@@ -85,10 +85,10 @@
...
 
 <body>
   <app-root></app-root>
-<script src="runtime.458556a34b891ea32398.js" defer></script><script src="polyfills-es5.0abfcb0e874b3127ca8b.js" nomodule defer></script><script src="polyfills.d152df52cf236990827a.js" defer></script><script src="main.6c4c6faefc370d823b85.js" defer></script></body>
+<script src="runtime.js" defer></script><script src="polyfills-es5.js" nomodule defer></script><script src="polyfills.js" defer></script><script src="styles.js" defer></script><script src="vendor.js" defer></script><script src="main.js" defer></script></body>
```
Comment (agyen) [inline dist/index.html]: white spaces???

**Pull request merged**